protos/flyteidl/admin/common.proto

```diff
@@ -21,10 +21,23 @@ message NamedEntityIdentifier {
     string name                  = 3;
 }
 
+// The status of the named entity is used to control its visibility in the UI.
+enum NamedEntityState {
+    // By default, all workflows are considered active and under development.
```

Comment:
nit on this comment being specific to workflows

Reply:
done